server/tests/taxonomy_term.rs

```diff
@@ -219,3 +219,63 @@ mod move_mutation {
         .await;
     }
 }
+
+#[cfg(test)]
+mod create_mutation {
+    use assert_json_diff::assert_json_include;
+    use test_utils::*;
+
+    #[actix_rt::test]
+    async fn creates_new_taxonomy_term() {
+        // TODO: maybe iterate over taxonomy types to test all?
```

Comment:
that would be great!

Reply:
yes, it would. 
The only trade-off is that the test gets quite slow, and even more if we test description as null.